general/development/process/peer-review/accessibility-checklist.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Accessibility checklist
+tags:
+  - Processes
+  - Integration
+  - Accessibility
+sidebar_position: 4
+---
+
+<!-- cspell:ignore ACTIVITYNAME, Deque -->
+
+This document proposes a checklist for accessibility reviews. Not all aspects of accessibility can be checked automatically, so this checklist is intended to guide reviewers when doing an accessibility check to a patch.
+
+## Identify the Nature of Changes
```

Comment:
```suggestion
:::tip Identify the Nature of Changes

- **Are the changes on the backend only (webservices, database updates, events, hooks...)?**<br/>
If yes, no accessibility review is needed.

- **Are there changes to the HTML, CSS, or JavaScript?**<br/>
If yes, an accessibility review is necessary.

- **Do the changes only involve standard HTML elements without altering default browser behaviour?**<br/>
If yes, a quick accessibility check is recommended, but a complete review may not be necessary.

:::
```

Reply:
I think this section would look better displayed as an admonition.

Reply:
But without the br tags

I wonder if this is a good fit for a definitinon list so that there is semantic markup instead of just bold

Reply:
I tried the list in pure HTMl using <dl>, but the result was not super clear without adding <strong> to the <dt> element. The final code in HTML is less friendly than a nested list alternative.
I changed it by markdown nested lists and the result is similar to what Sara was trying to do.